mijia-homie/src/config.rs

```diff
@@ -66,6 +68,17 @@ pub struct HomieConfig {
     pub device_name: String,
     pub prefix: String,
     pub sensor_names_filename: String,
+    /// The minimum time to wait between sending consecutive readings for the same sensor.
+    #[serde(
+        deserialize_with = "de_duration_seconds",
+        rename = "min_update_period_seconds"
+    )]
+    pub min_update_period: Duration,
```

Comment:
I was just thinking about interval vs period as names for this thing. I usually think of period for things that are regular, but if the period of updates is 6s and you set this to 10s then your updates will happen at:
0s
not 6s
12s
not 18s
... I guess periodic is correct. Ship it.